samza-kv-rocksdb/src/main/java/org/apache/samza/storage/kv/RocksDbKeyValueReader.java

```diff
@@ -80,6 +84,33 @@ public RocksDbKeyValueReader(String storeName, String dbPath, Config config) {
     }
   }
 
+  /**
+   * Construct the <code>RocksDbKeyValueReader</code> with store's descriptor,
+   * database's path and Samza's config
+   *
+   * @param descriptor describes rocksdb
+   * @param dbPath path to the db directory
+   */
+  public RocksDbKeyValueReader(RocksDbTableDescriptor descriptor, String dbPath) {
```

Comment:
Would it be simpler to use an in-memory store instead of RocksDB? Does RocksDB give something that in-memory does not?

Reply:
InMemory Store gets cleaned once the Samza job is finished, I spoke to @atoomula about it.